From 360dc7176ce04c312f4687a82d8338ef924aa16d Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Sun, 2 Aug 2020 20:17:19 +0200
Subject: [PATCH] Fix layout of tweets that contain an image and a cited tweet

Reverse the order of the image and the cited tweet. This also matches how these tweets are displayed on the Twitter web site

BUG: 424938
---
 libchoqok/ui/postwidget.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libchoqok/ui/postwidget.cpp b/libchoqok/ui/postwidget.cpp
index 971fd6f4..20da3b0c 100644
--- a/libchoqok/ui/postwidget.cpp
+++ b/libchoqok/ui/postwidget.cpp
@@ -88,7 +88,7 @@ const QString mImageTemplate(QLatin1String("<div style=\"padding-top:5px;padding
 
 const QLatin1String PostWidget::Private::resourceImageUrl("img://postImage");
 
-const QString PostWidget::baseTextTemplate(QLatin1String("<table height=\"100%\" width=\"100%\"><tr><td width=\"48\" style=\"padding-right: 5px;\">%1</td><td dir=\"%4\" style=\"padding-right:3px;\"><p>%2</p></td></tr>%6%5<tr><td></td><td style=\"font-size:small;\" dir=\"ltr\" align=\"right\" valign=\"bottom\">%3</td></tr></table>"));
+const QString PostWidget::baseTextTemplate(QLatin1String("<table height=\"100%\" width=\"100%\"><tr><td width=\"48\" style=\"padding-right: 5px;\">%1</td><td dir=\"%4\" style=\"padding-right:3px;\"><p>%2</p></td></tr>%5%6<tr><td></td><td style=\"font-size:small;\" dir=\"ltr\" align=\"right\" valign=\"bottom\">%3</td></tr></table>"));
 
 const QString PostWidget::baseStyle(QLatin1String("QTextBrowser {border: 1px solid rgb(150,150,150);\
 border-radius:5px; color:%1; background-color:%2; %3}\
-- 
GitLab

